DP02_09_Custom_Coadds/DP02_09b_Custom_Coadd_Sources.ipynb

```diff
@@ -10,7 +10,7 @@
     "<img align=\"left\" src = https://project.lsst.org/sites/default/files/Rubin-O-Logo_0.png width=250 style=\"padding: 10px\"> \n",
     "<b>Detect and Measure Sources in a Custom Coadded Image</b> <br>\n",
     "Contact author: Melissa Graham <br>\n",
```

Comment:
Add yourself as contact author to NB 09b.

Reply:
Done in commit [d07e509](https://github.com/rubin-dp0/tutorial-notebooks/pull/161/commits/d07e50984d027e042b936c92447c8ce710466be3).